addons/test_mail/tests/test_mail_gateway.py

```diff
@@ -1729,6 +1729,18 @@ def test_message_hebrew_iso8859_8_i(self):
             content
         )
 
+    def test_message_windows_874(self):
+        # Email for Thai customers who use Microsoft email service.
+        # The charset is windows-874 which isn't natively supported by
+        # python, check that Odoo is still capable of decoding it.
+        # windows-874 is the Microsoft equivalent of cp874.
+        with RecordCapturer(self.env['mail.test.gateway'], []) as capture:
```

Comment:
Why not using mock_mail_gateway, any reason or is it just a copy past of other test ?